db/seeds.rb

```diff
@@ -1,2 +1,59 @@
 # This file should contain all the record creation needed to seed the database with its default values.
 # The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
+# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
+ActiveRecord::Base.transaction do
+  AdminUser.where('username': 'admin').first_or_create!(
+    username: 'admin',
+    email: 'admin@domain.tld',
+    password: 'adminadmin',
+    password_confirmation: 'adminadmin',
+    identity_code: '38001085718',
+    country_code: 'EE',
+    roles: ['admin']
+  )
+  # Required for creating registrar
+  Setting.where('var': 'registry_vat_prc').first_or_create(
+    value: '0.2'
+  )
+  # First registrar
+  Registrar.where('name': 'Registrar First').first_or_create!(
```

Comment:
Why do you need to create a registrar? Given you create an `AdminUser` on line 5, he can login by itself and create registrars by himself. Unlikely anybody will benefit from such registrar.

Reply:
The idea is to have demo data set there to ease the testing of features in development

Reply:
`rake db:fixtures:load` should be used for this.

Reply:
Dummy data is already available in test/fixtures.